analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -219,7 +219,9 @@ class Cargo(
         // resolved and the dependency should not appear in the dependency tree.
         // See: https://doc.rust-lang.org/cargo/commands/cargo-metadata.html
         val dependency = node["dependencies"].find {
-            it.textValue().startsWith(dependencyName)
+            val text = it.textValue()
+            require(text.indexOf(' ') != -1) { "Unexpected format while parsing dependency JSON node." }
```

Comment:
Now that you already have the index, it would be slightly more elegant to also use it as the `endIndex` with `substring()`. Or, as another approach, you could use `split()`, require the length of the array to be `> 1`, and use the first array element.

Reply:
I think in this case our personal taste of elegance differ a bit. I prefer to keep it as is. Would you mind approving it?